src/workerd/io/compatibility-date.capnp

```diff
@@ -1227,4 +1227,18 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
   # When set, tells JSG to make the prototype of all jsg::Objects immutable.
   # TODO(soon): Add the default on date once the flag is verified to be
   # generally safe.
+
+  iterableTypeSupport @146 :Bool
```

Comment:
I believe the naming is too generic, getFetchBodyIterableSupport might be better?